README.md

```diff
@@ -1,17 +1,49 @@
 <p align="center">
   <a href="https://glide-core.crowdstrike-ux.workers.dev">
     <picture alt="Glide Core Logo">
-      <source srcset="https://github.com/CrowdStrike/glide-core/blob/main/.github/logos/light.png?raw=true" media="(prefers-color-scheme: light)" />
-      <source srcset="https://github.com/CrowdStrike/glide-core/blob/main/.github/logos/dark.png?raw=true" media="(prefers-color-scheme: dark)" />
-      <img src="https://github.com/CrowdStrike/glide-core/blob/main/.github/logos/light.png?raw=true" width="500" />
+      <source 
+        srcset="/.github/logos/light.png?raw=true" 
+        media="(prefers-color-scheme: light)" 
+      />
+
+      <source
+        srcset="/.github/logos/dark.png?raw=true"
+        media="(prefers-color-scheme: dark)"
+      />
+
+      <img src="/.github/logos/light.png?raw=true" width="500" />
     </picture>
+
   </a>
 </p>
 
 <p align="center"> 
   A Web Component design system from CrowdStrike.
 </p>
 
+<p align="center">
+  <a href="/src">
+    <img 
+      alt="Code coverage" 
+      src="https://img.shields.io/badge/coverage%20-%20100%25%20-%20yellowgreen" 
```

Comment:
Getting real-time coverage to show up is a whole thing and would require us to use a third-party service. I figured a hardcoded badge will do considering our thresholds.